streams/readable-streams/tee.any.js

```diff
@@ -317,6 +317,60 @@ promise_test(t => {
 
 }, 'ReadableStream teeing: erroring the original should immediately error the branches');
 
+promise_test(t => {
+
+  let controller;
+  const rs = new ReadableStream({
+    start(c) {
+      controller = c;
+    }
+  });
+
+  const [reader1, reader2] = rs.tee().map(branch => branch.getReader());
+  const cancelPromise = reader2.cancel();
+
+  controller.enqueue('a');
+
+  return flushAsyncEvents().then(() => {
```

Comment:
Why is it needed here and below?

Reply:
+1, `flushAsyncEvents()` should always come with a comment at least.

Reply:
You're right, it's not needed. I'll remove it.